build-cli.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```

Comment:
Thanks for point out. I’ll eventually turn this script in to ps1 and sh.

However, I find it’s nice to have one script across the platform. If we can agree on something that will be great.